src/App.tsx

```diff
@@ -5,6 +5,8 @@ import { routes, TradingLeague, Profile, LiquidityProvider, Trader, FixedBorrowe
 import { AlphaBanner, GweiBar } from '@components/composite';
 import Box from '@mui/material/Box';
 import { useEffect } from 'react';
+import useWallet from './hooks/useWallet';
```

Comment:
```suggestion
import { useWallet } from '@hooks';
```

lets keep same style throughout the code

Reply:
done in ff692256e500f567310f338bb4a684c73a01433a